app/queries/get_dependents.rb

```diff
@@ -0,0 +1,44 @@
+module Queries
+  module GetDependents
+    def self.call(content_id, link_type = nil)
+      results = ActiveRecord::Base.connection.execute(
+        query(content_id, link_type)
+      )
+
+      results.map { |r| r.fetch("content_id") }
+    end
+
+    def self.explain(content_id, link_type = nil)
+      ActiveRecord::Base.connection.execute(
+        "EXPLAIN #{query(content_id, link_type)}"
+      ).to_a
+    end
+
+    # Uses Postgres's WITH RECURSIVE feature to compute the transitive closure
```

Comment:
Can you please explain "transitive closure" in plainer English? This is a complicated query below and I (and future others) won't fully understand what it's doing and how it's doing it.